contracts/bprotocol/Registry.sol

```diff
@@ -136,6 +142,16 @@ contract Registry is Ownable {
         return _avatar;
     }
 
+    function _deployAvatarProxy(address _owner) internal returns (address proxy) {
+        bytes32 salt = keccak256(abi.encodePacked(_owner));
+        bytes memory deploymentData = abi.encodePacked(avatarProxyContractCode, uint256(avatarMaster));
```

Comment:
use `type(GnosisSafeProxy).creationCode` directly, and avoid expensive `sload`.

Reply:
is `uint256` intentional? (and not `address`)

Reply:
first comment fixed.

if we remove `uint256` we are getting revert. So its needed. I copied this code from GnosisSafeProxyFactory